frontend/src/utils/date.ts

```diff
@@ -160,3 +160,39 @@ export const transformCheckedBoxesValueToInvalidDays = (
   const validDaysSet = new Set(validDays)
   return ALL_INVALID_DAYS_ARR.filter((day) => !validDaysSet.has(day))
 }
+
+const DAY_TO_NUMBER_MAP: Record<InvalidDaysOptions, number> = {
+  [InvalidDaysOptions.Monday]: 1,
+  [InvalidDaysOptions.Tuesday]: 2,
+  [InvalidDaysOptions.Wednesday]: 3,
+  [InvalidDaysOptions.Thursday]: 4,
+  [InvalidDaysOptions.Friday]: 5,
+  [InvalidDaysOptions.Saturday]: 6,
+  [InvalidDaysOptions.Sunday]: 7,
+}
+
+/**
+ * Convert the days of the week in the invalidDays array
+ * to a number array representing the number representation
+ * of the corresponding day of the week
+ */
+export const convertInvalidDaysOfTheWeekToNumberSet = (
```

Comment:
same here. maybe we should make it a shared util if it is not already one

Reply:
I wanted to do it but couldn't find the shared file that both files could import from. But I found it now. I will abstract it out into a shared function.